hot-reload-analysis/src/main/kotlin/org/jetbrains/compose/reload/analysis/DirtyResolverExtension.kt

```diff
@@ -1,14 +0,0 @@
-/*
- * Copyright 2024-2025 JetBrains s.r.o. and Compose Hot Reload contributors.
- * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
- */
-
-package org.jetbrains.compose.reload.analysis
-
-import org.jetbrains.compose.reload.core.Context
-
-interface DirtyResolverExtension {
```

Comment:
This is part of the public API; I expect the CI to fail. I triggered a run for you. Please run tests as well!

Reply:
Finally I marked it as deprecated